pkg/controlplane/controller/defaultservicecidr/default_servicecidr_controller.go

```diff
@@ -184,33 +185,48 @@ func (c *Controller) syncStatus(serviceCIDR *networkingapiv1.ServiceCIDR) {
 	// don't sync the status of the ServiceCIDR if is being deleted,
 	// deletion must be handled by the controller-manager
 	if !serviceCIDR.GetDeletionTimestamp().IsZero() {
+		klog.V(4).Infof("ServiceCIDR %s is being deleted, skipping status sync", serviceCIDR.Name)
```

Comment:
4 is verbose mode in production for many. Shouldn't this be a bit higher?

Reply:
ack